modules/wmco-prerequisites.adoc

```diff
@@ -7,96 +7,58 @@
 
-== Supported platforms based on {product-title} and WMCO versions
+[id="wmco-prerequisites-supported-5.1.0_{context}"]
+== WMCO 5.1.0 supported platforms and Windows Server versions
 
-[cols="5",options="header"]
-|===
-|Platform
-|Supported {product-title} version
-|Supported WMCO version
-|Installer-provisioned infrastructure installation support
-|User-provisioned infrastructure installation support
-
-|Amazon Web Services (AWS)
-|4.6+
-|WMCO 1.0+
-|GA
-|
-
-|Microsoft Azure
-|4.6+
-|WMCO 1.0+
-|GA
-|
-
-|VMware vSphere
-|4.7+
-|WMCO 2.0+
-|GA
-|
-|===
-
-== Supported platforms for Bring-Your-Own-Host (BYOH) instances based on {product-title} and WMCO versions
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.1.0, based on the applicable platform. Windows Server versions not listed are not supported and will cause errors. To prevent these errors, use only an appropriate version for your platform.
 
-[cols="5",options="header"]
+[cols="3,7",options="header"]
 |===
 |Platform
-|Supported {product-title} version
-|Supported WMCO version
-|BYOH for installer-provisioned infrastructure installation support
-|BYOH for user-provisioned infrastructure installation support
+|Supported Windows Server version
 
 |Amazon Web Services (AWS)
-|4.8+
-|WMCO 3.1+
-|GA
-|
+|Windows Server 2019
 
 |Microsoft Azure
-|4.8+
-|WMCO 3.1+
-|GA
-|
+|Windows Server 2019
 
 |VMware vSphere
-|4.8+
-|WMCO 3.1+
-|GA
-|
-
-|Provider agnostic (Platform: none)
-|4.8+
-|WMCO 3.1+
-|
-|GA
+a|* Windows Server 2022 with the Windows link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637] patch.
+* Windows Server 20H2 
+
+|Bare metal/Provider agnostic
+a|* Windows Server 2022 with the Windows link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637] patch.
+* Windows Server 20H2 
 |===
 
-== Supported Windows Server versions
+[id="wmco-prerequisites-supported-5.0.0_{context}"]
+== WMCO 5.0.0 supported platforms and Windows Server versions
 
-The following table lists the supported link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server version] based on the applicable platform. Any unlisted Windows Server version is not supported and will cause errors. To prevent these errors, only use the appropriate version according to the platform in use.
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.0.0, based on the applicable platform. Windows Server versions not listed are not supported and will cause errors. To prevent these errors, use only the appropriate version for your platform.
```

Comment:
```suggestion
The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.0.0, based on the applicable platform. Windows Server versions not listed are not supported, and attempting to use them will cause errors. To prevent these errors, use only the appropriate version for your platform.
```